### ISSUE TITLE
Timezone settings in config.json

### ISSUE THREAD

**Issue by Hooch76** [kerberos-io/agent#82]
Hi,

I've set the timezone to `UTC+01:00 Amsterdam, Berlin, ... `

![image](https://user-images.githubusercontent.com/25639197/224316056-614c1531-4f41-49df-981f-019a197649b6.png)

and config.json shows `"timezone": "Arctic/Longyearbyen"`.

Is this expected?

Thanks.

**Comment by cedricve** [comment]
Yes, it's confusing but it will take the right timezone. Thanks for noticing this!
